src/main/cacheManager.ts

```diff
@@ -1,9 +1,21 @@
 import { Logger } from '@hmcts/nodejs-logging';
+import config from 'config';
+
+const redisPortalCredential = {
+  host: config.get('secrets.pip-ss-kv.REDIS_HOST'),
```

Comment:
these will need to be defined in the config/default.json (can even be blanks), otherwise app crashes when run